lighthouse-core/audits/bootup-time.js

```diff
@@ -34,6 +34,14 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
+// These trace events, when not triggered by a script inside a particular task, are just general Chrome overhead.
+const BROWSER_TASK_NAMES_SET = new Set([
+  'CpuProfiler::StartProfiling',
+  'V8.GCCompactor',
```

Comment:
IMO GC and `StartProfiling` are quite different things. The developer has some control over the amount of GC happening.

DevTools perf panel also has grouping by URL and when there's no URL it just says "unattributable". I think that makes sense for GC things. And any tracing/profiling overhead stuff i like putting into "Chrome" or "Browser"

Reply:
On second thought I remember seeing traces where there's a BIG GC at the beginning of a trace. Is that a problem you're solving here? If so I'm interested in a repro. I couldn't get one after trying a few things...

Reply:
Settled this as Browser, Browser GC, and Unattributable, but yeah important to keep in mind this is only for tasks that do not have any other `EvaluateScript`/timer/functioncall work. I *believe* this is mostly due to protocol overhead GC'ing. It's on the smaller end of the spectrum though